README.md

```diff
@@ -1,24 +1,163 @@
-# In Development - DO NOT USE
-
 # Train-Habitat
 
-Train Plugin for connecting to Habitat Supervisor HTTP Gateway
+`train-habitat` is a Train plugin and is used as a Train Transport to connect to Habitat installations.
 
 ## To Install this as a User
 
 You will need InSpec v2.3 or later.
 
 Simply run:
 
+```
 $ inspec plugin install train-habitat
+```
+
+## Using train-habitat from InSpec
+
+As train-habitat takes potentially many options, it is simplest to list the options in your `~/.inspec/config.json` file, then used the named set of options with `-t`.
+
+For example, if your config file contains:
+
+```
+{
+  "file_version": "1.1",
+  "credentials": {
+    "habitat": {
+      "dev-hab": {
+        "api_url": "http://dev-hab.my-corp.io",
+        "cli_ssh_host": "dev-hab.my-corp.io"
+      },
+      "prod-hab": {
+        "api_url": "https://prod-hab.my-corp.io",
+        "api_auth_token": "opensesame"
+      },
+    }
+  }
+}
+```
+
+Using this configuration, you could execute:
+
+```
+$ inspec exec some-profile -t habitat://dev-hab
+# Or
+$ inspec exec some-profile -t habitat://prod-hab
+```
+
+You may also pass `--config some-file.json` to use a config file at a different location.
+
+See below for the full list of options you may use with a `habitat` credential set in your configuration.
```

Comment:
Is there a particular section that "See below" refers to? Would it be possible to use a jump link for that?

Reply:
Yes; I'm always a bit shy when it comes to internal anchor links, because I often guess wrong about how the page renderer handles spaces in the anchor (assuming it inserts anchors).  I'll experiment.

Reply:
Totally understand. I know for .md that the internal link would be formatted as "[see below] (#header-name)" but I agree. I'm not quite sure how the page renderer treats such notation.